aip/client-libraries/4210.md

```diff
@@ -31,25 +35,90 @@ expectation.
 
 ## Guidance
 
+The general flow of code generation for client libraries and supporting code is
```

Comment:
maybe mention: "for most languages" (since e.g. in TypeScript / protobuf.js we only have GAPIC generator, but no stubs and no gRPC stubs)

Reply:
Good call, I called out node.js and python since they are different but both good examples of deviations